### ISSUE TITLE
Don't load js/css files that are not needed

### ISSUE THREAD

**Issue by slaFFik** [presscustomizr/hueman#184]
Example: 
```
http://example.com/wp-content/themes/hueman/assets/front/js/jquery.flexslider.min.js?ver=33cc2fbca9b35532269f411e8d836dad
```

Loaded on a front-page. But I don't use a build-in slider on a site. I believe this js file is used for featured posts slider. If this option is disabled - just don't load it.

**Comment by Nikeo** [comment]
yep good point.
there are several other improvements like this that I did not have the time to implement yet.
Like for example group the css stylesheets in one and minify it by default

**Comment by slaFFik** [comment]
```
http://example.com/wp-content/themes/hueman/assets/front/js/jquery.jplayer.min.js?ver=33cc2fbca9b35532269f411e8d836dad```

**Comment by Nikeo** [comment]
fixed in v3.0.12
